testcase/21.innerclasstest/src/test/InnerClassTest.java

```diff
@@ -0,0 +1,47 @@
+package test;
```

Comment:
src/main/java  and src/test/java should be source root folder, here you break the convention, see, https://github.com/vscjavaci/java-debug-ci/blob/master/testcase/5.breakpoint/src/main/java/BreakPointTest.java